doc/elements/tcsc.rst

```diff
@@ -0,0 +1,87 @@
+﻿.. _tcsc:
+
+=============
+Thyristor-Controlled Series Capacitor (TCSC)
+=============
+
+We implement the FACTS devices based on the PhD Thesis of Ara Panosyan, PhD.
```

Comment:
is there any link or name of the PhD Thesis? the reference is poor at the moment.